README.md

```diff
@@ -236,6 +236,60 @@ To do so you need to set `IDELOCALHOST` in you local.env file to the IP of your
 
 xdebug connections will then be sent to this IP address on port 9000.
 
+## Overriding / Extending
+
+You can add additional services, or modify current services, by creating a `docker-compose.override.yml` file. For example, to add a Redis service, add these contents to `docker-compose.override.yml`:
+
+``` yaml
+version: '3'
+services:
+  redis:
+    image: redis
+```
+
+To modify a current service, for example to specify volume caching for macOS:
+
+``` yaml
+version: '3'
+services:
+  web:
+  volumes:
+    - "${DOCKER_MW_PATH}:/var/www/mediawiki:cached"
+    - ./config/mediawiki:/var/www/mediawiki/.docker:ro
+    - ./config/hhvm/php.ini:/etc/hhvm/php.ini:ro
+    - ./config/hhvm/server.ini:/etc/hhvm/server.ini:ro
+    - ./scripts/wait-for-it.sh:/srv/wait-for-it.sh:ro
+    - mw-images:/var/www/mediawiki/images/docker
+```
+
+## Using VisualEditor and Parsoid
+
+Create a `docker-compose.override.yml` as documented above. In the override file, add:
+
+``` yaml
+version: '3'
+services:
+
+  parsoid:
+    image: thenets/parsoid:0.8.1
+    ports:
+      - "8000:8000"
+    environment:
+      - PARSOID_DOMAIN_mediawiki=http://default.web.mw.localhost/mediawiki/api.php
+      - PARSOID_LOGGING_LEVEL=debug
+    links:
+      - "web:default.web.mw.localhost"
+```
+
+In LocalSettings.php, add:
```

Comment:
If this is the only step required, I suppose it's fine to keep here, but generally I think MDD favours generalised approach with extension setup being deferred to the extensions' own documentation page(s).

I'd recommend doing both in this case, so that if/when someone has an issue, they know where to look for a potentially newer configuration and/or issues they might face.

Reply:
We could link to the mediawiki.org page for this step.